### PR TITLE
Fix slow tests v2

### PR DESCRIPTION
**Pull request by LysandreJik**
Fix a few tests that were broken in recent PRs.

### CODE REVIEW

File: tests/test_modeling_bart.py [hunk 1]
```diff
@@ -492,7 +492,9 @@ def test_mnli_inference(self):
         inputs_dict = prepare_bart_inputs_dict(model.config, input_ids)
         # Test that model hasn't changed
         with torch.no_grad():
-            batched_logits, features = model(**inputs_dict)
+            outputs = model(**inputs_dict)[:]
```
Comment (sgugger) [inline tests/test_modeling_bart.py:495]:
```suggestion
            outputs = model(**inputs_dict).to_tuple()
```
Might be easier toread like this?

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

File: tests/test_modeling_bart.py (hunk 1 above)
Comment (LysandreJik) [reply tests/test_modeling_bart.py:495]: Actually the `[:]` part wasn't supposed to be part of this PR. The `[0]` and `[1]` right below should be enough.

Comment (LysandreJik) [inline tests/test_modeling_bart.py:495]:
```suggestion
            outputs = model(**inputs_dict)
```

**Pull request merged**